README.md

```diff
@@ -3,8 +3,6 @@
 `dlordinal` is an open-source Python toolkit focused on deep learning with ordinal methodologies. It is compatible with
```

Comment:
`dlordinal` is not directly compatible with `scikit-learn`. However, all the methods implemented in `dlordinal ` can be used in classes that implement the `scikit-learn` estimator interface.